01_load_exported_data_and_visualize_pupillometry.ipynb

```diff
@@ -11,73 +11,49 @@
     "1. Visualize pupil diameter and pupil location as a function of time.\n",
```

Comment:
<p>Is there a reason for changing these?</p><p>The old tutorials use <code>0.95</code>.</p><p>Below we also still use <code>0.95</code>  for filtering.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pupil-labs/pupil-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>@pfaion - please revert this to <code>0.95</code>  and commit. </p>

Reply:
Done.